domains/authorization/Authorization.API/API.IntegrationTests/API.IntegrationTests/API/DeleteConsentTests.cs

```diff
@@ -0,0 +1,112 @@
+using API.IntegrationTests.Setup;
+using API.Models;
+using API.UnitTests;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.IntegrationTests.API;
+
+[Collection(IntegrationTestCollection.CollectionName)]
+public class DeleteConsentTests
```

Comment:
This tests what I believe should be implemented in this task.

Reply:
okay